tests/unit-core/test-exceptions.c

```diff
@@ -0,0 +1,175 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
```

Comment:
+1 to splitting the huge test-api.c into smaller test files.

I'd go even further and split this into test-abort.c and test-exec-stop.c.
At a glance they look like independent tests, no?

Reply:
For me that is a bit too much. Others?